toxcore/Makefile.inc

```diff
@@ -19,6 +19,8 @@ libtoxcore_la_SOURCES = ../toxcore/ccompat.h \
                         ../toxcore/ping_array.c \
                         ../toxcore/net_crypto.h \
                         ../toxcore/net_crypto.c \
+                        ../toxcore/net_profile.c \
```

Comment:
Is it intentional that you added these to this list twice?

Reply:
Nope, not sure how that happened.